### DISCUSSION TITLE
How to make MIDI and usb stdout work together?

### DISCUSSION THREAD

**Discussion by erfanium** [hathach/tinyusb#2808] [General]
I work an a midi project on a raspberry pi pico board. I want to also enable stout to be able to debug my code using `printf`.

But I can't make them work together. My device always will be recognized eaither as a midi device or tty device.

**Comment by hathach** [comment]
You need to add cdc interface to usb descriptor or Use pico debugger and output via uart

**Reply by erfanium** [reply]
Thanks! Yeah eventually i figure it out

here is the a sample repo if someone needs:
https://github.com/erfanium/tusb-cdc-midi-example
